app/components/DeleteButton/index.js

```diff
@@ -0,0 +1,34 @@
+/**
+*
+* DeleteButton
+*
+*/
+
+import React from 'react';
+
+import styles from './styles.css';
+
+function DeleteButton(props) {
+  return (
+    <button
+      className={styles.deleteButton}
+      onClick={() => {
+        const response = window.confirm(props.confirmText); // eslint-disable-line no-alert
+        if (response) {
+          props.onClick();
```

Comment:
I found this article through the MDN article about window.confirm: https://alistapart.com/article/neveruseawarning  
It basically advocates providing clear feedback and an obvious option to undo whatever was done instead of using a confirmation box. I like that idea, although we'd need to consider return on (time) investment.